caterva2/services/sub.py

```diff
@@ -89,25 +88,27 @@ def __init__(self, abspath, path, metadata=None):
             if self.abspath is not None:
                 self.abspath.parent.mkdir(exist_ok=True, parents=True)
 
-    def _get_request_args(self, nchunk):
+    def _get_client_and_request_args(self, nchunk, return_async_client):
```

Comment:
I'm not a big fan of long function names.  I suggest to keep the previous name, or e.g. `_get_client_args`, but if you feel strongly about this, do your thing, no need to fight ;-)

Reply:
done